docs/content/en/use-cases/spread-insert.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Spread `INSERT`s among cluster shards
+title: Spread INSERTs among cluster shards
```

Comment:
Could keep it if was reflected on the website but the backticks propage to `og:title`meta.

Reply:
Hmm... backticks shouldn't propagate to the og:title, we should fix that. Not sure we can fix it in our repo, maybe we'll need to file a PR upstream. I'll add an issue about this.